_templates/_page_openshift.html.erb

```diff
@@ -127,10 +127,10 @@
   <script src="https://assets.openshift.net/content/modernizr.js" type="text/javascript"></script>
   <script src="https://assets.openshift.net/content/subdomain.js" type="text/javascript"></script>
   <script src="https://assets.openshift.net/content/nav-tertiary.js" type="text/javascript"></script>
-  <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/bootstrap-offcanvas.js" type="text/javascript"></script>
-  <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/reformat-html.js" type="text/javascript"></script>
-  <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/hc-search.js" type="text/javascript"></script>
-  <script src="https://docs.openshift.com/container-platform/4.1/_javascripts/page-loader.js" type="text/javascript"></script>
+  <script src="/container-platform/4.2/_javascripts/bootstrap-offcanvas.js" type="text/javascript"></script>
```

Comment:
This won't work for docs.okd.io or (I think) ARO docs. Further I prefer 4.1 as that is a reference point (first release for 4.x). I understand that not doing this makes it harder to test locally but hard coding the location of the scripts was the only way I could make it work at 4.1 release time with changes to our asciibinder script. Sorry! :(

Reply:
Fair point and thanks for catching this. I have reverted changes to this file, keeping the original hard-coded locations of the scripts. I have also used the same style (absolute 4.1 path) in order to source the script in `search-commercial.html` for consistency.